third_party/render_globals.go

```diff
@@ -4,6 +4,7 @@ import (
 	"bytes"
 	"log"
 	"text/template"
+	"github.com/googlecodelabs/tools/claat/util"
```

Comment:
Alphabetical order for imports

Reply:
Per other files in this repo, imports follow alphabetical order, but standard library imports and our own packages are separate. Missed a new line here